### DISCUSSION TITLE
make verify-kube-connect fails

### DISCUSSION THREAD

**Discussion by Gaikanomer9** [argoproj/argo-cd#5367] [Q&A] [answered]
Hello!
I want to run the connectivity check and get the following error:
```
bash: kubectl: command not found
make: *** [verify-kube-connect] Error 127
```

As I understand the kubectl command is passed to the Docker container and the binary kubectl file should be available inside?
My config for the kubectl:
```
which kubectl
/usr/local/bin/kubectl
kubectl version
Client Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-13T13:28:09Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"darwin/amd64"}
Server Version: version.Info{Major:"1", Minor:"20", GitVersion:"v1.20.2", GitCommit:"faecb196815e248d3ecfb03c680a4507229c2a56", GitTreeState:"clean", BuildDate:"2021-01-21T01:11:42Z", GoVersion:"go1.15.5", Compiler:"gc", Platform:"linux/amd64"}
go version
go version go1.15.6 darwin/amd64
```
And I'm using the Kind cluster and have the valid IP address in the kube config (not localhost). 

I've tried pulling the image and building it but no luck so far. What could be the issue here?

**Comment by jannfis** [comment, marked as answer]
This seems to be a collateral damage from https://github.com/argoproj/argo-cd/pull/5101 and is definitely a bug, which probably also affects other developer targets in the Makefile.

Will send a fix soon.

/cc @iam-veeramalla

**Reply by jannfis** [reply]
When #5369 is merged, it should work again.

**Reply by Gaikanomer9** [reply]
Thank you @jannfis!
